### ISSUE TITLE
Feature - PostgreSQL Support 

### ISSUE THREAD

**Issue by z1pti3** [b1scuit-thi3f/jimiPlugin-database#2]
Not an issue but a feature requests for future if possible to include PostgreSQL as an option?

**Comment by b1scuit-thi3f** [comment]
Makes sense - I should probably also add MongoDB support and a few others...
